integration-cli/docker_cli_run_test.go

```diff
@@ -3363,7 +3363,7 @@ func TestRunRestartMaxRetries(t *testing.T) {
 		t.Fatal(string(out), err)
 	}
 	id := strings.TrimSpace(string(out))
-	if err := waitInspect(id, "{{ .State.Restarting }} {{ .State.Running }}", "false false", 5); err != nil {
+	if err := waitInspect(id, "{{ .State.Restarting }} {{ .State.Running }}", "false false", 10); err != nil {
```

Comment:
cause I was running test and I think because laptop was under heavy load (compiling kernel) and this tests timed out, I don't know if this was the problem, test runs ok now, I can change it back